File: taller php/operaciones/operacion.php
<?php
// Solicitar el valor de A y B
echo "Ingrese valor A: ";
$A = (int)trim(fgets(STDIN));

echo "Ingrese valor B: ";
$B = (int)trim(fgets(STDIN));

// Sumar los números
$suma = $A + $B;
echo "La suma de los números es: $suma\n";

// Restar los números
$res = $A - $B;
echo "La resta de los números es: $res\n";

// Multiplicar los números
$multi = $A * $B;
echo "La multiplicación de los números es: $multi\n";

// Dividir los números
if ($B != 0) {
    $div = $A / $B;
    echo "La división de los números es: $div\n";
} else {
    echo "No se puede dividir por 0\n";
}

// Comparar si A es igual a B
$igual = ($A == $B);
echo "El número A es igual a B: " . ($igual ? "Sí" : "No") . "\n";

// Comparar si A es mayor que B
$mayor = ($A > $B);
echo "El número A es mayor que B: " . ($mayor ? "Sí" : "No") . "\n";

// Comparar si A es menor que B
$menor = ($A < $B);
echo "El número A es menor que B: " . ($menor ? "Sí" : "No") . "\n";
?>

File: talleres c++/operacion.cpp
#include <iostream>
using namespace std;

int main() {
    int A, B;  // Declaramos las variables A y B para los valores de entrada

    // Solicitar al usuario que ingrese los valores de A y B
    cout << "Ingrese valor para A: ";
    cin >> A;

    cout << "Ingrese valor para B: ";
    cin >> B;

    // Realizar las operaciones
    int suma = A + B;
    cout << "La suma de los numeros es: " << suma << endl;

    int res = A - B;
    cout << "La resta de los numeros es: " << res << endl;

    int multi = A * B;
    cout << "La multiplicacion de los numeros es: " << multi << endl;

    // Dividir, mostrando el resultado como float
    float div = static_cast<float>(A) / B;
    cout << "La division de los numeros es: " << div << endl;

    // Comparaciones
    bool igual = (A == B);
    cout << "Los numeros son iguales: " << igual << endl;

    bool mayor = (A > B);
    cout << "El numero mayor es: " << (mayor ? A : B) << endl;
    cout << "El numero menor es: " << (mayor ? B : A) << endl;

    return 0;
}
